docker-compose.yml

```diff
@@ -11,11 +11,18 @@ services:
       - 80
     volumes:
       - .:/application/code:rw
+  pgadmin:
+    image: thajeztah/pgadmin4
+    ports:
+      - "5050:5050"
```

Comment:
I'd recommend just using `5050` here and letting `docker-compose` allocate the local port to avoid conflicts with other application docker-compose files that might also be making use of `pgadmin`.